example/lib/home_controller.dart

```diff
@@ -16,9 +16,7 @@ class HomeController extends GetxController {
   var message = TextEditingController();
   var phoneNumber = TextEditingController();
   var browserClientWebSocketUrl = TextEditingController();
-  String? get browserEndPoint => browserClientWebSocketUrl.text.isNotEmpty
-      ? browserClientWebSocketUrl.text
-      : null;
+  String? get browserEndPoint => browserClientWebSocketUrl.text.isNotEmpty ? browserClientWebSocketUrl.text : null;
```

Comment:
@loic-hamdi please revert these changes and make sure to use dart formatting , to keep consistency